CONTRIBUTING.md

```diff
@@ -3,14 +3,24 @@
 Follow these steps **only once**:
 1. Go to https://github.com/campa-consortium/lattice-standard.
 
-2. Click the pulldown arrow next to `Fork`, click `Create a new fork`, and specify your GitHub username in the `Owner` field (e.g., "cemitch99").
+2. Click the pulldown arrow next to `Fork`, click `Create a new fork`, and specify your GitHub username in the `Owner` field (e.g., "username").
 
-3. Click `Code` to view and copy the URL of your fork 
-(e.g., https://github.com/cemitch99/lattice-standard.git)
+3. Go back to https://github.com/campa-consortium/lattice-standard and copy the url of the repo:
+(e.g., https://github.com/campa-consortium/lattice-standard.git)
```

Comment:
This is a bit of an individual contributor choice, but there will be contributors who prefer to clone via SSH rather than HTTPS, in which case the link would be `git@github.com:campa-consortium/lattice-standard.git`. Maybe we could simplify this instruction and simply say "Clone the repository on your local computer" or something along these lines, without being too detailed? Just a suggestion, up to you all.

Reply:
Actually, point 4. is about cloning the repository, so maybe we could just skip point 3. and assume that contributors know where to find the link to clone the repository.

Reply:
I think it doesn't hurt to be as detailed and explicit as possible, even if this is only helpful to 1-2 people; those who prefer to clone via SSH should know how to modify this accordingly. What do you think?

Reply:
@cemitch Thoughts?

Reply:
I originally used HTTPS exactly because it was _easier_--for me it worked entirely without worrying about authentication and tokens.  I agree SSH is a better choice, provided we anticipate questions about permissions and authentication.  I will change this here.

Reply:
The text should note that if a person just wants to look at the repo, HTTPS is easier since HTTPS does not require the password setup.

PS: I just discovered that a person has to be logged into GitHub to see the SSH option after pressing the `code` button.